core/src/epicli/cli/engine/ansible/AnsibleRunner.py

```diff
@@ -70,6 +70,11 @@ def pre_flight(self, inventory_path):
                                                        playbook_path=self.playbook_path('preflight'),
                                                        retries=1)
 
+        self.logger.info('Setting up SSH tunnel for Kubectl')
+        self.ansible_command.run_playbook_with_retries(inventory=inventory_path,
```

Comment:
Not sure if this is used there as at this stage on first run there will not be a K8s cluster jet.

Reply:
So the preflight_facts role is used to make our repository work in AWS where we use autoscaling groups. Otherwise adding a new node can basically change ordering in inventory and make the cluster unusable. But this only makes some sense when kubernetes_master group is populated. So I guess to make it simple we can skip kubectl refactor in this role. 🤔

Reply:
So I'll revert those changes for this role. Now LGTM.

Reply:
How abut setting up SSH tunnel for kubectl conditionally? Only for self-managed K8s.

Reply:
It is beeing checking in the Ansible code.